### PR TITLE
Add interface to get all menus of a role

### PR DESCRIPTION
**Pull request by wu-clan**
please review.

### CODE REVIEW

File: backend/app/common/casbin_rbac.py [hunk 1]
```diff
@@ -43,18 +43,24 @@ async def rbac_verify(self, request: Request, _: dict = DependsJwtAuth) -> None:
...
             return
         if settings.MENU_PERMISSION:
             # 菜单权限校验
             path_auth = request.url.path.replace(f'{settings.API_V1_STR}', '').replace('/', ':')
             menu_perms = []
+            forbid_menu_perms = []
             for role in user_roles:
-                menu_perms.extend([menu.perms for menu in role.menus])
+                for menu in role.menus:
+                    menu_perms.append(menu.perms) if menu.status == 1 else forbid_menu_perms.append(menu.perms)
```
Comment (downdawn) [inline backend/app/common/casbin_rbac.py:58]: Status can use enumerated types

File: backend/app/api/v1/menu.py [hunk 2]
```diff
@@ -35,6 +35,11 @@ async def get_all_menus(
     menu = await MenuService.get_menu_tree(title=title, status=status)
     return await response_base.success(data=menu)
 
+@router.get('/{pk}/all', summary='获取角色所有菜单', dependencies=[DependsJwtAuth])
```
Comment (downdawn) [inline backend/app/api/v1/menu.py:38]: GET /roles/{role_id}/menus
more appropriate?

**Review by downdawn — APPROVED** [review]

File: backend/app/common/casbin_rbac.py (hunk 1 above)
Comment (wu-clan) [reply backend/app/common/casbin_rbac.py]: updated

File: backend/app/api/v1/menu.py (hunk 2 above)
Comment (wu-clan) [reply backend/app/api/v1/menu.py]: updated

**Review by downdawn — APPROVED** [review]

**Pull request merged**